### DISCUSSION TITLE
How many backtests at once before ban?

### DISCUSSION THREAD

**Discussion by orlagwon** [akumidv/tradingview-assistant-chrome-extension#179] [Q&A]
I know for sure that 2 backtests per account will never lead to ban.

Please, share your experience on how many browsers windows can run backtest for one account without getting banned.

**Comment by Sc4ramanga** [comment]
i can confirm that 2 are possible. to me it seems that deep backtesting is the real issue for the Tradingview ppl. got a time limit ban due to have usage only with deep mode
